app/views/dashboard/pages/home.html.erb

```diff
@@ -1 +1,84 @@
-<h>Welcome to Dashboard Home</h1>
\ No newline at end of file
+<section class="section" id="content">
+  <!-- Statistics -->
+  <div class="container">
+    <h3 class="title is-5">Statistics</h3>
+      <div class="columns">
+        <div class="column">
+            <div class="box statistic">
+                <div class="media ">
+                    <div class="media-left image is-48x48">
+                            <h3 class="icon title is-2 has-text-primary is-block"><i class="fas fa-folder-plus"></i></h3>
+                    </div>
+                    <div class="media-content ">
+                        <p class="subtitle is-6">My Giveaways</p>
+                        <h4 class="title is-4 "><%= current_user.giveaways.count %></h4>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="column">
+            <div class="box statistic">
+                <div class="media ">
+                    <div class="media-left image is-48x48">
+                        <h3 class="icon title is-2 has-text-link is-block"><i class="fas fa-file-alt"></i></h3>
+                    </div>
+                    <div class="media-content ">
+                        <p class="subtitle is-6">Giveaways Given</p>
+                        <h4 class="title is-4 "><%= @given_giveaways.count %></h4>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="column">
+            <div class="box statistic">
+                <div class="media ">
+                    <div class="media-left image is-48x48">
+                            <h3 class="icon title is-2 has-text-dark is-block"><i class="fas fa-paste"></i></h3>
+                    </div>
+                    <div class="media-content ">
+                        <p class="subtitle is-6">My Claims</p>
+                        <h4 class="title is-4 "><%= current_user.claims.count %></h4>
```

Comment:
Can we do here `@claims.count` ?

Reply:
@Yes, we have the instance variable. Tnx